src/Stats/index.jsx

```diff
@@ -1,33 +1,103 @@
-        {utils.sortDataByDate(transactions).map((item, index) => (
-          <ListItem
-            key={`ListKey${index + 1}`}
-            title={item.label ? item.label : 'Undefined'}
-            subtitle={moment.unix(item.date).format('MM/DD/YYYY')}
-            rightTitle={
-              item.type === 'Income'
-                ? ` + $${item.amount}`
-                : ` - $${item.amount}`
-            }
-            leftIcon={{
-              title: 'Unknown',
-              name: 'attach-money',
-            }}
-          />
-        ))}
-      </View>
-    </ScrollView>
+    <View style={{ flex: 1, flexDirection: 'column' }}>
+      <Overlay height={200} isVisible={overlay}>
+        <View>
+          <View style={{ flexDirection: 'row', justifyContent: 'flex-end' }}>
+            <Button icon={{ name: 'close' }} type="clear" onPress={updateOverlay} />
+          </View>
+          <View style={{ marginTop: 20 }}>
+            <ListItem title="Month" topDivider bottomDivider onPress={() => updateHeaderView('month')} />
+            <ListItem title="Year" bottomDivider onPress={() => updateHeaderView('year')} />
+          </View>
+        </View>
+      </Overlay>
+      <MainHeader title="Activity" onPressBtn={updateOverlay} />
+      <SubHeader viewSet={view === 'month' ? currentMonthSet : currentYearSet} onPressBtn={updateCurrent} viewType={view} />
+      <ScrollView style={styles.deviceBody}>
+        <View style={{ flexDirection: 'row', justifyContent: 'space-between' }}>
+          <LargeButton subtitle="Spent" title="-$400.00" note="Expense" bgColor="red" btnAmount={2} />
+          <LargeButton subtitle="Earned" title="+$680.00" note="Incomes" bgColor="green" btnAmount={2} />
+        </View>
+        {ifTransition(transactions)
+          ? updateListView(utils.groupData(transactions)).map((value, index) => (
+            <View style={styles.card} key={`DateGroup:${index + 1}`}>
+              <View style={styles.cardAlign}>
+                <Text style={styles.cardHeader}>
+                  {value[0].date}
+                </Text>
+                <Text style={styles.cardHeader}>
+                  {utils.sumAmount(value)}
+                </Text>
+              </View>
+              {value.map((item, idx) => (
+                <ListItem
+                  key={`InfoGroup:${idx + 1}`}
+                  leftElement={<GradientIcon name={item.label ? item.label.icon : 'home'} color={item.label && item.label.color ? item.label.color : '#9e87fc'} iconFamily={item.label && item.label.iconFamily ? item.label.iconFamily : ''} />}
+                  title={item.label ? item.label.name : 'Unknown'}
+                  subtitle={item.comment ? item.comment : 'Unknown'}
+                  subtitleStyle={{ color: 'grey' }}
+                  rightTitle={item.type === 'Expense' ? `-$${item.amount}` : `+$${item.amount}`}
+                />
+              ))}
+
+            </View>
+          )) : <View><Text>NULL</Text></View>}
```

Comment:
what is this NULL?

Reply:
It is a view which displays while there's no transaction records, moved the content into a new component called EmptyTrans, will do the styling later

Reply:
But we shouldn't display NULL to the user, that is not user frinedly at all, it looks like an error in the app. Let's say, "You haven't added any transaction yet"

Reply:
Okay, have updated the information